### ISSUE TITLE
Decide on whether to launch with pubkey or addr proofs

### ISSUE THREAD

**Issue by lsankar4033** [personaelabs/nym#4]
spartan-ecdsa proofs of sig->pubkey are considerably cheaper in terms of prover complexity than sig->address (b/c keccak)

the difference in UX is huge; minute proving time vs. 3s proving time

ideally we'd launch with proof of sig->pubkey but not all nouner addresses have pubkeys associated (i.e. b/c haven't submitted a tx). in the long-term, we can have a form input for signing a transaction and sending pubkey, but in the near-term, it'd be simplest to just use sig->pubkey proofs

this only makes sense if most of the pseudonymous discourse is initially only from addresses with a pubkey

this issue tracks making the decision here. current thinking is to manually look through vocal nouner addresses and determine how many are held at addresses with pubkeys

**Comment by lsankar4033** [comment]
looked through the data for owners (past/present) who don't have an address:
- https://dune.com/ls43/nouns-pubkey-checks/6fad3627-3509-4202-9ee8-25699e5780ca
- https://github.com/lsankar4033/eth_analyses/tree/main/packages/noun_pubkeys

tl;dr - used the heuristic of 'owns an ENS name' as a proxy for governance participation. all of the no-tx addresses with an ENS name are gnosis SAFEs, adn all of these had at least one owner that has a tx

final decision: let's go with proof of pubkey<>sig and have a submission modal for those who aren't supported to submit a signed tx. for a later launch including those pubkeys as well (cc @DanTehrani @amirgamil @cha0sg0d )

**Comment by lsankar4033** [comment]
> > have a submission modal for those who aren't supported to submit a signed tx. for a later launch including those pubkeys as well
> 
> @lsankar4033 can you expand a bit on what you mean here?

ah yes! 

if a user for whom we don't have a pubkey gets to the nym creation interface, they a message like "we don't currently support nyms for users who haven't made transactions on-chain. to be included in a later release, sign this message" and a button that signs a message via web3 wallet + sends it to our server along with address

one day, if enough people do this, we can release v2 with all of their pubkeys included as well
